etc/config/functional.suite.dist.yml

```diff
@@ -14,18 +14,13 @@ modules:
         - \Magento\FunctionalTestingFramework\Module\MagentoWebDriver
         - \Magento\FunctionalTestingFramework\Helper\Acceptance
         - \Magento\FunctionalTestingFramework\Helper\MagentoFakerData
-        - \Magento\FunctionalTestingFramework\Module\MagentoRestDriver:
-            url: "%MAGENTO_BASE_URL%/rest/default/V1/"
-            username: "%MAGENTO_ADMIN_USERNAME%"
-            password: "%MAGENTO_ADMIN_PASSWORD%"
-            depends: PhpBrowser
-            part: Json
         - \Magento\FunctionalTestingFramework\Module\MagentoSequence
         - \Magento\FunctionalTestingFramework\Module\MagentoAssert
         - Asserts
     config:
         \Magento\FunctionalTestingFramework\Module\MagentoWebDriver:
             url: "%MAGENTO_BASE_URL%"
+            backend_url: "%MAGENTO_BACKEND_BASE_URL%"
```

Comment:
I think this changes should be reflected in Jenkins infra repo
https://github.com/magento-cicd/jenkins-infra/blob/master/mftf/mdc/functional.suite.yml.chrome

Reply:
Very good call. Builds are running fine without it, but I'll create a PR to update the above.